src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -82,26 +80,13 @@ export const MyFieldAndBackground = () => {
   return (
     <Box
       sx={{
-        display: 'grid',
-        gridTemplateColumns: {
-          xs: '1fr',
-          md: '3fr 1fr',
-        },
-        columnGap: '1rem',
-        rowGap: '1rem',
-        gridTemplateAreas: {
-          xs: '"field-and-background" "research-profile"',
-          md: '"field-and-background research-profile"',
-        },
+        bgcolor: 'secondary.main',
       }}>
-      <BackgroundDiv
-        sx={{
-          bgcolor: 'info.light',
-        }}>
-        <Formik initialValues={initialValues} onSubmit={(values) => updatePerson.mutate(values)} enableReinitialize>
-          {({ isSubmitting, dirty, setFieldValue, resetForm }: FormikProps<PersonBackgroundFormData>) => (
-            <Form>
-              <Box>
+      <Formik initialValues={initialValues} onSubmit={(values) => updatePerson.mutate(values)} enableReinitialize>
+        {({ isSubmitting, dirty, setFieldValue, resetForm }: FormikProps<PersonBackgroundFormData>) => (
+          <Form>
+            <Box sx={{ width: '80%', ml: '1rem', mt: '1rem' }}>
```

Comment:
Litt uklart hvorfor vi ønsker 80% bredde uansett skjermstørrelse, men alltid 1rem margin til venstre. Ser litt ut som at det er for å få ca 1rem margin til høyre også på mobil?
Hvis casen er at vi ønsker maksbredde på input-feltet er det kanskje bedre å sette max-width med rem eller noe sånt, sånn at det tar hele bredden å mobil og bare cappes når skjermen blir brei nok til at det blir et problem?

```suggestion
            <Box sx={{ mx: '1rem', mt: '1rem' }}>
```

Reply:
Satt `maxWidth` på container. Tenkte å ta opp ideen om en maxWidth på hovedinnholdet (som vi snakket om i forrige uke) på designmøtet på onsdag, så kanksje vi får etablert en standard på det,